crypto/fipsmodule/aes/asm/aes-gcm-avx2-x86_64.pl

```diff
@@ -446,19 +446,16 @@ sub _ghash_4x {
     my ( $GHASH_ACC_PTR, $HTABLE, $AAD, $AADLEN ) = @argregs[ 0 .. 3 ];
 
     # Additional local variables
-    my ( $TMP0,       $TMP0_XMM )       = ( "%ymm0", "%xmm0" );
-    my ( $TMP1,       $TMP1_XMM )       = ( "%ymm1", "%xmm1" );
-    my ( $TMP2,       $TMP2_XMM )       = ( "%ymm2", "%xmm2" );
-    my ( $LO,         $LO_XMM )         = ( "%ymm3", "%xmm3" );
-    my ( $MI,         $MI_XMM )         = ( "%ymm4", "%xmm4" );
-    my ( $GHASH_ACC,  $GHASH_ACC_XMM )  = ( "%ymm5", "%xmm5" );
-    my ( $BSWAP_MASK, $BSWAP_MASK_XMM ) = ( "%ymm6", "%xmm6" );
-    my ( $GFPOLY,     $GFPOLY_XMM )     = ( "%ymm7", "%xmm7" );
-    my $H_POW2_XORED = "%ymm8";
-    my $H_POW1_XORED = "%ymm9";
+    my $TMP0_XMM       = "%xmm0";
+    my $TMP1_XMM       = "%xmm1";
+    my $TMP2_XMM       = "%xmm2";
+    my $LO_XMM         = "%xmm3";
+    my $GHASH_ACC_XMM  = "%xmm4"; # Different than upsream
+    my $BSWAP_MASK_XMM = "%xmm5"; # Different than upsream
+    my $GFPOLY_XMM     = "%xmm6"; # Different than upsream
```

Comment:
Note: If we can avoid xmm6 then we can avoid a memory store in the prologue and a memory read in the epilogue on Windows.